### PR TITLE
next swap

### PR DESCRIPTION
**Pull request by shendel**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/CONTRIBUTING) guide
- [x] Styleguide check `npm run validate`
- [x] Good naming, keep simple
- [x] Tested desktop/mobile
- [x] Tested bright/dark
- [x] Tested en/ru
- [x] Affects money; I checked the functionality once again
- [x] I checked the PR once again

### Original issue

#

### CODE REVIEW

**Comment by noxonsu** [comment]
https://github.com/swaponline/MultiCurrencyWallet/issues/3394

**Comment by iower** [comment]
Tested, swapped. All works fine, except of infinite NEXT balance loading after a swap (<kbd>Check balance</kbd>).

File: src/core/swap.swaps/integration/BtcLikeSwap.js [hunk 1]
```diff
@@ -0,0 +1,701 @@
...
+  }
+
+  _initSwap(app) {
+    super._initSwap(app)
+
+    this.app = app
+
+    /* init networks if need */
+    const {
+      main: {
```
Comment (noxonsu) [inline src/core/swap.swaps/integration/BtcLikeSwap.js:71]: переименовать в mainnet и testnet

File: src/front/shared/helpers/getPairFees.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  COIN_DATA,
+  COIN_MODEL,
+  COIN_TYPE,
+} from 'swap.app/constants/COINS'
+import { BigNumber } from "bignumber.js"
+import helpers from 'helpers'
+
+
+console.log('COIN_DATA', COIN_DATA)
+const fetchCoinFee = (coin) => {
```
Comment (noxonsu) [inline src/front/shared/helpers/getPairFees.js:11]: вынести из фронта в общий файл

File: src/core/swap.swaps/integration/BtcLikeSwap.js [hunk 3]
```diff
@@ -0,0 +1,701 @@
...
+
+        throw new Error(`txinfo={confirmations: ${confirmations}, fees: ${fees}, size: ${size}, senderAddress: ${senderAddress} }`)
+
+      } catch (err) {
+        console.error(`BtcSwap: Error fetching confidence: using confirmations > 0:`, err.message)
+        return confidenceFromConfirmations
+      }
+    }
+
+    const confidences = await Promise.all(unspents.map(fetchConfidence))
```
Comment (noxonsu) [inline src/core/swap.swaps/integration/BtcLikeSwap.js:166]: confidences  - поменять название, не понятно что это

**Comment by noxonsu** [comment]
- [ ] https://screenshots.wpmix.net/chrome_xjfVZW5AImIBkhsTjnlXy1JEBg0wEy1x.png

**Review by noxonsu — APPROVED** [review]

**Pull request merged**